plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -80,19 +80,9 @@ def sync_repo(self, repo, sync_conduit, config):
         """
         working_dir = tempfile.mkdtemp(dir=repo.working_dir)
         try:
-            try:
-                # This will raise NotImplementedError if the config's feed_url is determined not to
-                # support the Docker v2 API.
-                self.sync_step = sync.SyncStep(repo=repo, conduit=sync_conduit, config=config,
-                                               working_dir=working_dir)
-            except NotImplementedError:
-                # Since the feed_url was determined not to support the Docker v2 API, let's use the
-                # old v1 SyncStep instead.
-                self.sync_step = v1_sync.SyncStep(repo=repo, conduit=sync_conduit, config=config,
-                                                  working_dir=working_dir)
-
+            self.sync_step = sync.SyncStep(repo=repo, conduit=sync_conduit, config=config,
```

Comment:
@asmacdo Unfortunately, this would prevent us from having one progress report per sync operation, which would not be compatible with Pulp's API.